xbmc/system_egl.h

```diff
@@ -0,0 +1,50 @@
+/*
+ *  Copyright (C) 2005-2020 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#pragma once
+
+#if defined(TARGET_LINUX)
+
+#if defined(WL_EGL_PLATFORM)
+#undef WL_EGL_PLATFORM
+#define KODI_WL_EGL_PLATFORM
+#endif
+
+#if defined(__GBM__)
+#undef __GBM__
+#define KODI__GBM__
+#endif
+
+#if defined(USE_X11)
+#undef USE_X11
+#define KODI_USE_X11
+#endif
+
+#define EGL_NO_X11
```

Comment:
Don't we have to define `MESA_EGL_NO_X11_HEADERS` too for older mesa?

Reply:
updated thanks :)